src/App/components/Sidebar/SidebarSearchResults/OrdersSearchResults.tsx

```diff
@@ -85,16 +94,52 @@ export default function OrdersSearchResults(props: propsIF) {
     };
 
     return (
-        <div>
-            <h6 className={styles.card_title}>My Limit Orders</h6>
+        <FlexContainer
+            flexDirection='column'
+            justifyContent='center'
+            alignItems='flex-start'
+            gap={8}
+        >
+            <Text fontWeight='500' fontSize='body' color='accent5'>
+                My Limit Orders
+            </Text>
             {searchedLimitOrders.length ? (
-                <>
-                    <header className={styles.header}>
-                        <div>Pool</div>
-                        <div>Price</div>
-                        <div>Value</div>
-                    </header>
-                    <ol className={styles.main_result_container}>
+                <FlexContainer flexDirection='column' fullWidth>
```

Comment:
So this is something I don't love - we managed to turn 6 lines into like 35 lines.  

Seems like a weird thing bc this is the whole reason you'd use classes. You don't want to do the same inline styling for 3 of the same components. As a dev I'd much rather just create the text component I want and then uses it 3 times. or however many is needed.